Monika After Story/game/script-topics.rpy

```diff
@@ -8960,6 +8960,217 @@ label monika_hydration:
+    m 1eku "Yet, here we are, in planet Earth."
+    m 1hub "Isn't that great? You should be feeling lucky right now just for being alive!"
+    m 2euu "It's almost like life itself gave us the oportunity to live..."
+    m 3esb "You better remember that next time you are feeling down, [player]!"
+    m 3tsb "Or... You could also come to me and allow me to lift your mood~"
+    m 1hublb "Ehehe~"
+    return
+
+define persistent._mas_didplayerbetray = None
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_betrayal",category=['misc'],prompt="Betrayal",random=True))
+
+label monika_betrayal:
+    m 1eka "...Say, [player]?"
+    m 1lka "Something came to my mind right now."
+    m 1eka "Have you ever betrayed someone by any means?"
+    m 4eka "I don't mean to 'betray' little promises with friends that could even be considered jokes."
+    m 4eku "I mean it in a real way... You know, breaking the confidence someone had with you because you did something terrible to them."
+    m "You don't have to answer if you don't want to, it's completely understandable."
+    $ _history_list.pop()
+    menu:
+        m "You don't have to answer if you don't want to, it's completely understandable."
+        "Yes, I have betrayed someone.":
+            $ persistent._mas_didplayerbetray = True
+            m 2ekc "..."
+            m "It's alright, [player]"
+            m 3ekc "I know you probably didn't mean to do that."
+            m 3ekb "We all mess up sometimes, don't we?"
+            m 3esb "It's not like I'm going to hate you for what you did, you silly~"
+
+        "No, I haven't betrayed anyone.":
+            $ persistent._mas_didplayerbetray = False
+            m 2huu "I'm happy to hear that, [player]."
+            m 3efb "People's trust isn't something to be played with!"
+            m 3esa "And just for letting you know..."
+            m 3hublb "I...{w=2}will never betray you!~"
+
+        "...":
+            m 2ekc "It's alright, [player]."
+            m "I won't force you to say anything."
+            m 3ekc "What kind of girlfriend would I be if I did that?"
+            m 3ekb "Ehehehe...Let's just forget this topic existed in first place."
+    return
+        
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_photography",category=['misc'],prompt="Photography",random=True))
+
+label monika_photography:
+    m 1eua "...Say, [player]?"
+    m 1esa "Do you happen to know about photography?"
+    m 3esu "It has become an interestic topic for me."
+    m "The more I research about it the more I like it."
+    m 3lsu "For example, did you know that the first photograph was taken using a box with a hole in it as a camera?"
+    m "Lenses weren't introduced until much later."
+    m 3esb "Also, the firsts mechanisms of photography reveal were based on a series of special liquids were the picture needed to be introduced during a certain amount of time"
```

Comment:
Did you mean "where" instead of "were"?

Missing a period. 

I think this line is a bit long and could be shortened or broken up to be less confusing.

Reply:
Changed the line to make it less confusing.